test/cql-pytest/conftest.py

```diff
@@ -21,6 +21,9 @@
 import time
 import random
 
+import sys
+sys.path.insert(1, sys.path[0] + '/../..')
```

Comment:
See above

Reply:
To make it a package the directory should be renamed to cql_pytest, python have a regex that not allow directories with '-' to be a package. But such proposal was declined by the owner https://github.com/scylladb/scylladb/pull/11395